ToDo/ToDo/View/TaskTableViewCell.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  TaskTableViewCell.swift
+//  ToDo
+//
+//  Created by Amirzhan Armandiyev on 06.11.2022.
+//
+
+import UIKit
+
+class TaskTableViewCell: UITableViewCell {
+    static let identifier = "TaskCell"
+    let taskLabel: UILabel = {
+        let label = UILabel()
+        label.textColor = .black
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.numberOfLines = 0
+        label.textAlignment = .center
+        return label
+    }()
+    let addressLabel: UILabel = {
+        let label = UILabel()
+        label.textColor = .black
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.numberOfLines = 0
+        label.textAlignment = .center
+        return label
+    }()
+    let dateLabel: UILabel = {
+        let label = UILabel()
+        label.textColor = .black
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.numberOfLines = 0
+        label.textAlignment = .center
+        return label
+    }()
+    
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        
+        contentView.addSubview(taskLabel)
```

Comment:
You can add 2 separate methods: `addSubviews()` and `configureConstraints()` and call them in init